<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1079</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Partorium - Electronic Parts Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>14</y>
      <width>181</width>
      <height>610</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_left">
     <item>
      <widget class="QLineEdit" name="lne_Search"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbb_Category"/>
     </item>
     <item>
      <widget class="QListWidget" name="lst_Parts"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>311</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Infos</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>90</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="labelAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_ID">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_IDValue">
        <property name="text">
         <string>IDValue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Category">
        <property name="text">
         <string>Kategorie:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_CategoryValue">
        <property name="text">
         <string>CategoryValue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_Subcategory">
        <property name="text">
         <string>Unterkategorie:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_SubcategoryValue">
        <property name="text">
         <string>SubCategoryValue</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Quantity">
        <property name="text">
         <string>Menge:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lbl_QuantityValue">
        <property name="text">
         <string>AmountValue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_ShortDescriptionValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Kurzbeschreibung</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_PartNameValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>11</y>
      <width>581</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PartName</string>
    </property>
   </widget>
   <widget class="QListView" name="lst_Images">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>589</width>
      <height>61</height>
     </rect>
    </property>
    <property name="viewMode">
     <enum>QListView::ViewMode::IconMode</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Image">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>20</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>250</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>250</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: white;
	border: 1px solid #444444; /* Dunkelgrau */
}</string>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="grp_Storage">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>80</y>
      <width>269</width>
      <height>81</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>100</height>
     </size>
    </property>
    <property name="title">
     <string>Lagerort</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>42</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_Storage">
        <property name="text">
         <string>Lagerort:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_StorageValue">
        <property name="text">
         <string>LagerortValue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_StorageDetailsValue">
        <property name="text">
         <string>DetailsValue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_StorageDetails">
        <property name="text">
         <string>Details:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_Files">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>269</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Dateien</string>
    </property>
    <widget class="QListWidget" name="lst_Files">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>251</width>
       <height>301</height>
      </rect>
     </property>
     <property name="resizeMode">
      <enum>QListView::ResizeMode::Adjust</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_Links">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>269</width>
      <height>91</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>120</height>
     </size>
    </property>
    <property name="title">
     <string>Links</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>251</width>
       <height>66</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="labelAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_Source">
        <property name="text">
         <string>Quelle:    </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_SourceValue">
        <property name="text">
         <string>QuelleValue</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_AlternativeSource">
        <property name="text">
         <string>Alternativ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_AlternativeSourceValue">
        <property name="text">
         <string>AltQuelleValue</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_Manufacturer">
        <property name="text">
         <string>Hersteller:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lbl_ManufacturerLink">
        <property name="text">
         <string>HerstellerValue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lbl_Hashtags">
    <property name="geometry">
     <rect>
      <x>201</x>
      <y>621</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Hashtags:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_HashtagsValues">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>621</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HashtagsValue</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_Description">
    <property name="geometry">
     <rect>
      <x>202</x>
      <y>351</y>
      <width>589</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1079</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="act_NewPart"/>
    <addaction name="separator"/>
    <addaction name="act_SelectDatabase"/>
    <addaction name="act_OpenDatabaseFolder"/>
    <addaction name="separator"/>
    <addaction name="act_OpenSettingsDialog"/>
    <addaction name="separator"/>
    <addaction name="act_Quit"/>
   </widget>
   <widget class="QMenu" name="menuWerkzeuge">
    <property name="title">
     <string>Werkzeuge</string>
    </property>
    <addaction name="actionHTML_bersicht"/>
   </widget>
   <widget class="QMenu" name="menuAnsicht">
    <property name="title">
     <string>Ansicht</string>
    </property>
    <addaction name="act_ShowDeletedParts"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuWerkzeuge"/>
   <addaction name="menuAnsicht"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_NewPart">
   <property name="text">
    <string>Neues Bauteil</string>
   </property>
  </action>
  <action name="act_Quit">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionHTML_bersicht">
   <property name="text">
    <string>HTML Übersicht</string>
   </property>
  </action>
  <action name="act_SelectDatabase">
   <property name="text">
    <string>Datendatei wählen</string>
   </property>
  </action>
  <action name="act_OpenDatabaseFolder">
   <property name="text">
    <string>Datendatei im Finder/Explorer anzeigen</string>
   </property>
  </action>
  <action name="act_OpenSettingsDialog">
   <property name="text">
    <string>Einstellungen</string>
   </property>
  </action>
  <action name="act_ShowDeletedParts">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gelöschte Bauteile anzeigen</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>act_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>354</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
